libs/payments/ui/src/lib/client/components/SubscriptionContent/en.ftl

```diff
@@ -29,3 +29,14 @@ subscription-content-button-cancel-subscription =
 subscription-content-button-cancel =
   Cancel
   .aria-label = Cancel your subscription to { $productName }
+subscription-content-cancel-action-error = An unexpected error occurred. Please try again.
+
+subscription-cancellation-dialog-title = We’re sorry to see you go
+# $name (String) - The name of the subscribed product.
+# $date (Date) - Last day of product access
+subscription-cancellation-dialog-msg =
```

Comment:
Very minor nit: unless there's a reason for it, could we keep the string values on the same line as the ID?
(It's perfectly valid syntax, it just makes it harder to scan when reviewing.)